extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -504,37 +509,26 @@ export class AmpA4A extends AMP.BaseElement {
   /** @override */
```

Comment:
not sure where this lines up but hopefully fixed when I explicitly added catch to promise chain initialization in onLayoutMeasure